ci-operator/step-registry/openshift-qe/installer/OWNERS

```diff
@@ -3,6 +3,7 @@ approvers:
 - paigerube14
```

Comment:
If this folder OWNERS file is the same as the file in the openshift-qe main folder think we should have a symbolic link here so we dont need to update every sub folder every time we have an edit to the owners

Example: https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/cluster-density-v2/OWNERS

Reply:
@paigerube14 Updated, thanks!